InnerEye/ML/Histopathology/datamodules/base_module.py

```diff
@@ -87,17 +100,20 @@ def prepare_data(self) -> None:
             self._load_dataset(self.val_dataset, stage='val', shuffle=True)
             self._load_dataset(self.test_dataset, stage='test', shuffle=True)
 
-    def _dataset_pickle_path(self, stage: str) -> Optional[Path]:
+    def _dataset_pt_path(self, stage: str) -> Optional[Path]:
```

Comment:
what's PT? maybe spell out?

Reply:
it is the standard abbreviation for pytorch when saving in pickle format - I can revert to pickle if more clear?

Reply:
revert to pickle for more clarity